telebot/service_utils.py

```diff
@@ -0,0 +1,85 @@
+import random
+import string
+
+try:
+    # noinspection PyPackageRequirements
+    from PIL import Image
+    from io import BytesIO
```

Comment:
No ideas why problems wirh this import are expected and threaten as no PIL installed.

Probably this import should be done outside of try block

Reply:
PIL is not enlisted in package requirements, so it's the same as with redis for StateRedisStorage